quizapp/src/main/java/billennium/quizapp/entity/User.java

```diff
@@ -0,0 +1,31 @@
+package billennium.quizapp.entity;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.SequenceGenerator;
+import javax.persistence.Table;
+
+@Entity
+@Table(name = "USER")
+@AllArgsConstructor
+@NoArgsConstructor
+@Data
+public class User {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.SEQUENCE, generator="user_id_seq")
+    @SequenceGenerator(name="user_id_seq", sequenceName="user_id_seq", allocationSize=1)
+    @Column(name = "ID")
+    private Long id;
+
+    @Column(name = "NAME")
```

Comment:
Deklaracja `name` w adnotacji `@column` jest potrzebna tylko wtedy gdy chcemy aby kolumna miała inną nazwe niż pole w javie. Gdy sa takie same adnotacja jest niepotrzebna. Tyczy się to linijki 25 i 28

Reply:
> ok